dev/domain-controller/new-admin-vm/Install-CAM.ps1

```diff
@@ -332,7 +332,7 @@ Configuration InstallCAM
 				while ($retryCount -gt 0)
 				{
 					$readyToConfigure = ( Get-Item $JVMServerdll -ErrorAction SilentlyContinue )
-					# don't wait for {64A3A4F4-B792-11D6-A78A-00B0D0180910} - that's the JDK. The JRE is installed 2nd {26A...} so wait for that.
+					# don't wait for {64A3A4F4-B792-11D6-A78A-00B0D0180144} - that's the JDK. The JRE is installed 2nd {26A...} so wait for that.
```

Comment:
Looks like this comment was duplicated in error? (Probably by me previously...)

Reply:
it was there previously, so I just updated the value

Reply:
But does it make sense to keep it?

Reply:
No.  I can remove it